Schweizerisches Bundesgericht.js

```diff
@@ -0,0 +1,243 @@
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+const abbrevPublished = {
+	de: "BGE ",
+	fr: "ATF ",
+	it: "ATF "
+};
+
+const abbrevUnpublished = {
+	de: "BGer ",
+	fr: "TF ",
+	it: "TF "
+};
+
+const courtName = {
+	de: "Schweizerisches Bundesgericht",
+	fr: "Tribunal Fédéral Suisse",
+	it: "Tribunale Federale Svizzero"
+}
+
+/**
+ * read query parameters from url
+ *
+ * @param   {String} url
+ * @returns {{}}
+ */
+function parseURLParameters(url) {
+	let parts = require('url').parse(url, true);
+	parts.query._lang = parts.pathname.substring(21, 23);
+
+	let subpath = parts.pathname.substring(23);
+	if (subpath === "/php/aza/http/index.php") {
+		parts.query._collection = "BGer";
+	}
+	else if (subpath === "/php/clir/http/index.php") {
+		parts.query._collection = "BGE";
+	}
+	else if (subpath === "/php/aza/http/index_aza.php") {
+		parts.query._collection = "news";
+	}
+	
+	return parts.query;
+}
+
+/**
+ * internal detectWeb interpreting prepared query parameters
+ *
+ * @param   {{}} parameters
+ * @returns {string|boolean}
+ * @private
+ */
+function _detectWeb(parameters) {
+	switch (parameters.type) {
+		case "simple_query":
+		case "simple_similar_documents":
+			return "multiple";
+
+		case "show_document":
+		case "highlight_simple_query":
+		case "highlight_simple_similar_documents":
+			return "case";
+	}
+	if (parameters.mode === "news") {
+		return "multiple";
+	}
+	return false;
+}
+
+/**
+ * collect items from a new rulings page
+ *
+ * Those pages are found at:
+ * /ext/eurospider/live/de/php/aza/http/index_aza.php?...mode=news
+ *
+ * @param {Document} doc
+ * @param {{}}       parameters
+ * @returns {{}}
+ */
+function getSearchResultsNews(doc, parameters) {
+	let domRows = doc.querySelectorAll("div.eit>table:nth-of-type(2)>tbody>tr");
+	let items = {};
+
+	for (let i = 1; i < domRows.length; i += 2) {
+		let date = domRows[i].children[1].innerText;
+		let url  = domRows[i].children[2].children[0];
+		let number = domRows[i].children[2].children[0].innerText;
+		let subject = domRows[i].children[4].innerText;
+		items[url] = number + " (" + date + "; " + subject + ")";
+	}
+
+	return items;
+}
+
+/**
+ * collect items from a search result page of all rulings
+ *
+ * Those pages are found at:
+ * /ext/eurospider/live/de/php/aza/http/index.php?l...type=simple_query|simple_similar_documents
+ *
+ * @param {Document} doc
+ * @param {{}}       parameters
+ * @returns {{}}
+ */
+function getSearchResultsBGer(doc, parameters) {
+	let domRows = doc.querySelectorAll("div.ranklist_content>ol>li");
```

Comment:
```suggestion
	let domRows = doc.querySelectorAll(".ranklist_content ol > li");
```

For resilience to site changes.